examples/ex5p.cpp

```diff
@@ -326,7 +326,19 @@ int main(int argc, char *argv[])
                       DataCollection::PARALLEL_FORMAT);
    visit_dc.Save();
 
-   // 16. Send the solution by socket to a GLVis server.
+   // 16. Save data in the Paraview format
+   ParaviewDataCollection paraview_dc("PWExample5", pmesh);
+   paraview_dc.SetLevelsOfDetail(1);
+   paraview_dc.SetCycle(1);
+   paraview_dc.SetTime(0.0);
+   paraview_dc.RegisterField("velocity",u);
+   paraview_dc.RegisterField("pressure",p);
+   paraview_dc.Save();
+
```

Comment:
Too many unnecessary newlines.

Reply:
I am not sure that I understand the  comment - These are the lines demonstrating the functionality of the object.  I can flag them within an if statement - let me know if it will be better.

Reply:
Sorry for not being clear enough, the paraview demonstration is fine. I just meant to remove the empty lines following the demonstration code block in order to reduce the total number of lines in the file.

Reply:
Corrected :-)